packages/@office-iss/react-native-win32/overrides.json

```diff
@@ -279,22 +279,25 @@
       "baseHash": "9b194f61db28366f5fb490b033d940636a9c7bb2"
     },
     {
-      "type": "copy",
-      "file": "src/Libraries/Lists/__tests__/FillRateHelper-test.js",
+      "type": "patch",
```

Comment:
Why are all of these now different than upstream?

Reply:
In upstream they upgraded the prettier version but it looks like some of the formatting updates were not applied to these files leaving our files different from theirs. Unclear why some of the prettier updates were not present.